CondTools/Ecal/src/EcalTPGWeightIdMapHandler.cc

```diff
@@ -58,11 +58,11 @@ void popcon::EcalTPGWeightIdMapHandler::getNewObjects() {
 	//check whats already inside of database
 	if (tagInfo().size){
   	//check whats already inside of database
-	  edm::LogInfo("got offlineInfo = ");
-	  edm::LogInfo("tag name = ") << tagInfo().name;
-	  edm::LogInfo("size = ") << tagInfo().size;
+    	std::cout << "got offlineInfo = " << std::endl;
```

Comment:
Why the edm::LogInfo ( which is the correct way to implement logging ) has been turned to std::cout  statements?